docs/secure-connections/sauce-connect/troubleshooting.md

```diff
@@ -141,6 +141,14 @@ Another common mistake is placing the SUT in the same network as the [Demilitari
 | SUT (Site Under Test)              | The site that you're testing.                                            |
 | Tunnel VM (Tunnel Virtual Machine) | Virtual machine that hosts Sauce Connect Proxy on the Sauce Labs side.   |
 
+## Known Issues and Workarounds
+
+### HTTP and Chrome 120 and newer
+
+Chrome 120 automatically attempts to upgrade connections from HTTP to HTTPS, and the error detection fails when using a proxy. In an interactive session, a second attempt falls back to HTTP, however there is no workaround when controlling the browser through an automated test.
```

Comment:
great. I'd add a little more detail

Reply:
Added a sentence about how SC uses a proxy, and why the proxy returns an error that confuses Chrome. WDYT?